libmscore/note.cpp

```diff
@@ -1395,7 +1395,7 @@ bool Note::readProperties(XmlReader& e)
             while (e.readNextStartElement()) {
                   const QStringRef& t(e.name());
                   if (t == "Event") {
-                        NoteEvent ne;
+                        NoteEvent ne = NoteEvent();
```

Comment:
What is the purpose of this change? `NoteEvent ne;` already invokes `NoteEvent`'s default constructor.

Reply:
Oh right